common/djangoapps/student/admin.py

```diff
@@ -326,35 +327,54 @@ class LoginFailuresAdmin(admin.ModelAdmin):
     """Admin interface for the LoginFailures model. """
     list_display = ('user', 'failure_count', 'lockout_until')
     raw_id_fields = ('user',)
-    search_fields = ('user__username', 'user__email')
+    search_fields = ('user__username', 'user__email', 'user__first_name', 'user__last_name')
     actions = ['unlock_student_accounts']
     change_form_template = 'admin/student/loginfailures/change_form_template.html'
 
-    def has_pem(self, request, method):
+    class _Feature(object):
```

Comment:
What is the purpose of this class? Wouldn't it be more readable to use only a static method here? Or maybe we could just move this to a function in `common/djangoapps/student/helpers.py`?

Reply:
That sounds good to me. Thanks for the extensive explanation!

Reply:
FWIW, I believe you could also extract it into a function above this model if the class vs. function thing matters to you. But I'm fine with it being where it is.